indicators/NO_BFLY_001/R/NO_BFLY_001.qmd

```diff
@@ -0,0 +1,652 @@
+  quiet = TRUE)
+# remove marine areas
+regions <- sf::st_intersection(regions, nor)
+
+tm_shape(regions) + 
+  tm_polygons(col = "region") 
+
+```
+
+## 8. Spatial units
+
+```{=html}
+<!--# 
+
+Describe the spatial units that you rely on in your analyses. Highlight the spatial units (the resolution) that the indicator values should be interpretted at. Potential spatial delineation data should eb introduced under 7.1. Datasets. We recomend using the SEEA EA terminology opf Basic Spatial Units (BSU), Ecosystem Asses (EA) and Ecosystem Accounting Area (EAA). 
+
+-->
+```
+
+Ecosystem assets: Mid-, central-, east-, and southern Norway. Note that southern Norway only includes the old counties of Vestfold and Østfold.
+
+Ecosystem accounting area: Norway (although N-Norway is missing).
+
+Data needs to be aggregated to EA level before interpretation. In addition, only locations with complete sampling (three replicated per season) should be included to avoids seasonal biases.
+
+## 9. Analyses
+
+```{=html}
+<!--# 
+
+Use this header for documenting the analyses. Put code in seperate code chunks, and annotate the code in between using normal text (i.e. between the chunks, and try to avoid too many hashed out comments inside the code chunks). Add subheaders as needed. 
+
+Code folding is activated, meaning the code will be hidden by default in the html (one can click to expand it).
+
+Caching is also activated (from the top YAML), meaning that rendering to html will be quicker the second time you do it. This will create a folder inside you project folder (called INDICATORID_cache). Sometimes caching created problems because some operations are not rerun when they should be rerun. Try deleting the cash folder and try again.
+
+-->
+```
+
+These calculations with bootstrap sampling are implemented in the R-package [bombLepiSurv](https://github.com/NINAnor/bombLepiSurv).
+
+Install the bombLepiSurv-package like this
+
+```{r}
+#| eval: false
+#| code-fold: show
+remotes::install_github("NINAnor/bombLepiSurv")
+library(bombLepiSurv)
+```
+
+Connect to the database, download maps and plot
+
+```{r connect}
+# get secret username and password
+source(here::here("_pw.R"))
+# connect
+bombLepiSurv::humlesommerfConnect(
+  username = BOMB_USER,
+  password = BOMB_PASSWORD,
+  host = "ninradardata01.nina.no"
+)
+```
+
+```{r fig-butterfly-regions}
+#| fig-cap: 'Map of regions in Norway coverede by the NMBB program. Østlandet is only covered by two fylker, and northern Norway is not covered at all'
+# Get maps
+nbbm_norway_regions <- bombLepiSurv::get_map()
+
+# Filer and plot
+norway_reg_NBBM2 <- nbbm_norway_regions |> 
+  filter(region %in% c("Sør", "Øst", "Vestlandet", "Trøndelag")) |>
+  select(region) %>% 
+  group_by(region) %>% 
+  summarize(geom = st_union(geom))
+
+tm_shape(norway_reg_NBBM2) + 
+  tm_polygons(col = "region") 
+```
+
+Download data from 2023 as an example. 2024 data is not yet published.
```

Comment:
note that the 2024 is available in the "new_database_structure" branch of bombLepiSurv. I can try to get it.

Reply:
I wond to this now. It would ba a quite manual hack.